tests/main/interfaces-joystick/task.yaml

```diff
@@ -0,0 +1,57 @@
+summary: Ensure that the joystick interface works.
+
+details: |
+    The joystick interface allows reading and writing to joystick devices.
+
+prepare: |
+    . "$TESTSLIB/snaps.sh"
+    install_local test-snapd-joystick
+
+    . "$TESTSLIB/files.sh"
+    ensure_file_exists_backup_real /dev/input/js31
+    ensure_file_exists_backup_real /run/udev/data/c13:31
+    ensure_file_exists_backup_real /run/udev/data/c13:67
+    ensure_file_exists_backup_real /dev/input/event67
+
+restore: |
+    rm -f call.error
+
+    . "$TESTSLIB/files.sh"
+    clean_file /dev/input/js31
```

Comment:
And this?

Reply:
This is removing the file and in case there is an original file it is restored.